### PR TITLE
Transparency fix

### PR DESCRIPTION
**Pull request by gavrilovmiroslav**

### CODE REVIEW

File: src/Loaders.cpp [hunk 1]
```diff
@@ -30,67 +28,9 @@ memory::RawPtr<Texture> TextureLoader::load_texture(String path)
...
-	}
-
-	Logger::info("Updating texture");
-	SDL_UpdateTexture(texture->inner, nullptr, pixels, width * sizeof(U32));
-
-	Logger::info("Freeing texture storage");
-	free(pixels);
-
+	auto* surface = IMG_Load(path.c_str());
+	texture->inner = SDL_CreateTextureFromSurface(state.renderer, surface);
```
Comment (drage0) [inline src/Loaders.cpp:32]: `IMG_Load` creates a surface, but it's never freed. Is this intentional? Am I missing something? The surface should be freed once the texture is created.

**Review by Mentathiel — CHANGES_REQUESTED** [review]
Separate transparency fix and assets into two PRs or remove the assets completely if they're there just for testing.

File: src/Loaders.cpp (hunk 1 above)
Comment (gavrilovmiroslav) [reply src/Loaders.cpp:32]: Probably not, this sounds sane, I'll check!

**Comment by harry-cpp** [comment]
> Separate transparency fix and assets into two PRs

Unlike perforce, you can separate unrelated changes in one PR, you just need to split it into 2 commits :)

**Comment by gavrilovmiroslav** [comment]
I'm having problems with my local computer and meson certification, so I'm pushing this with the hope that it builds and passes the automatic checks...

**Review by Mentathiel — APPROVED** [review]

**Pull request merged**